### PR TITLE
fixed the nav

### PR DESCRIPTION
**Pull request by Webkinz1690**
nav is now routing

### CODE REVIEW

File: client/src/pages/FAQs.jsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
         <h2 className="h1-responsive font-weight-bold text-center my-4 blue-text">
           Ready to Wyn it?
         </h2>
         <p className="lead grey-text w-responsive text-center mx-auto mb-5">
-          Our super streamlined interface, 
-          coupled with easy-to-use functionality makes it the most effective productivity 
-          timer out there. Don't waste time trying to figure it out, <br></br> just press <strong>start</strong>.
+          Our super streamlined interface, coupled with easy-to-use
+          functionality makes it the most effective productivity timer out
+          there. Don't waste time trying to figure it out, <br></br> just press{' '}
```
Comment (ErnieAtLYD) [inline client/src/pages/FAQs.jsx:15]: You could probably remove the `{' '}`. Also, `<br></br>` is a little bit of a pet peeve of mine. Could you just wrap stuff in a div tag and add margin instead if you need the white space?

File: server/routes/secure/users.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const express = require('express'),
   User = require('../../db/models/user'),
   multer = require('multer'),
   sharp = require('sharp'),
-  { sendCancellationEmail } = require('../../emails');
+  { sendCancellationEmail } = require('../../emails/index');
```
Comment (ErnieAtLYD) [inline server/routes/secure/users.js:6]: I _think_ that so long as you have an `index.js` in that directory you should be okay just doing `require('../../emails/')`. But you may want to double-check that first just to make sure.

File: client/src/pages/FAQs.jsx (hunk 1 above)
Comment (Webkinz1690) [reply client/src/pages/FAQs.jsx:15]: If i removed the {' '} there is no space between the <strong>start</strong> and the rest of the text. I'll change the margins though.

File: server/routes/secure/users.js (hunk 2 above)
Comment (Webkinz1690) [reply server/routes/secure/users.js:6]: Done. Thank you.

**Pull request merged**